roles/dtc/common/templates/ndfc_vrf_lite_ospf.j2

```diff
@@ -0,0 +1,80 @@
+! This NDFC VRF-Lite BGP config data structure is auto-generated
+! DO NOT EDIT MANUALLY
+!
+!
+{% if item.ospf.bfd.enabled %}
+feature bfd
+!
+{% endif %}
+vrf context {{ item.vrf }}
+  address-family ipv4 unicast
+    route-target both auto
+    route-target both auto evpn
+!
+{% if switch_item.redistribution is defined and switch_item.redistribution %}
+  {% for switch_redist in switch_item.redistribution %}
+    {% if switch_redist.source == 'bgp' and switch_redist.route_map %}
+route-map {{ switch_redist.route_map }} permit 10
+  match route-type internal
+!
+router ospf {{ item.ospf['process'] }}
+  vrf {{ item.vrf }}
+    redistribute bgp {{ MD_Extended.vxlan.global.bgp_asn }} route-map {{ switch_redist.route_map }}
+!
+    {% endif %}
+    {% if switch_redist.source == 'ospf' and switch_redist.route_map %}
+route-map {{ switch_redist.route_map }} permit 10
```

Comment:
Remove this when ready for formal review

Reply:
Removed. Done